src/_zkapauthorizer/tests/test_model.py

```diff
@@ -856,6 +865,72 @@ def test_lease_maintenance_activity(self, get_config, now, activity):
         )
 
 
+class EventStreamTests(TestCase):
+    """
+    Tests related to the event-stream storage of VoucherStore
+    """
+
+    @given(
+        tahoe_configs(),
+        posix_safe_datetimes(),
+        lists(sql_identifiers(), min_size=1),
+        tables(),
+        data(),
+        lists(sampled_from([inserts, deletes, updates]), min_size=1),
+    )
+    def test_event_stream_serialization(
+        self, get_config, now, ids, table, data, change_types
+    ):
+        """
+        Various kinds of SQL statements can be serialized into and out of
+        the event-stream.
+        """
+        # no BLOBs
+        assume(
+            not any(
+                column[1].affinity == StorageAffinity.BLOB for column in table.columns
+            )
+        )
```

Comment:
For what it's worth, SQLite3 will also quote these for us with `quote()` so we could support this if we wanted.  I don't think we have any blobs in the schema right now but we could be storing a slightly more compact representation of the cryptographic tokens if we did.

Since we don't need blobs now, change here is certainly optional - but I think blob support in `quote_sql_value` is fewer lines than this assumption.

Reply:
Okay, it wasn't clear to me that `quote()` would also work for BLOBs _and_ I figured we weren't using them etc .. but I can just let it through if you think that's better.